website/docs/d/container_registry.markdown

```diff
@@ -0,0 +1,42 @@
+---
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_container_registry"
+sidebar_current: "docs-azurerm-datasource-image"
+description: |-
+  Get information about an Image
```

Comment:
can we make this `Use this data source to access information about a Container Registry.` to match the comment below?